src/views/authors/detail.pug

```diff
@@ -32,7 +34,10 @@ block content
           else
             p #{t('authors.noSongs')}
 
+    if error_msg.length
+      #errorAlert.alert.alert-danger #{error_msg}
     .mt-4
       a.btn.btn-primary(href=`/admin/authors/update/${author.id}`) #{t('common.edit')}
       a.btn.btn-danger(href=`/admin/authors/delete/${author.id}`) #{t('common.delete')}
```

Comment:
Kiểm tra nếu author có songs thì không xóa được => cho button delete mờ đi (or cho màu xám) và không click được ý (cursor: not-allowed)

Reply:
thì hiển thị ra thông báo là có song nên không xóa được cũng được mà pà